webapp/src/Controller/Jury/ContestController.php

```diff
@@ -259,7 +260,7 @@ public function indexAction(Request $request): Response
                 }
             }
 
-            if ($this->isGranted('ROLE_ADMIN')) {
+            if ($this->isGranted('ROLE_ADMIN') && !$contest->isLocked()) {
```

Comment:
Might be cooler to grey them out in this case?

Reply:
I actually have a slight preference to not show buttons you can't click.

Reply:
I have the same preference, but:
In case that we auto lock contests (in the future) and someone doesn't know about this new feature, that person would never find the edit buttons but when you grey it out it's clear that its currently not possible (and we could even have a remark when you hover over it)

Reply:
That's a good point, let's keep it in mind when we add the auto lock configuration